src/integrationTest/java/uk/gov/hmcts/reform/waworkflowapi/config/OpenAPIPublisherTest.java

```diff
@@ -17,15 +17,15 @@
  * Built-in feature which saves service's swagger specs in temporary directory.
  * Each travis run on master should automatically save and upload (if updated) documentation.
  */
-class SwaggerPublisherTest extends SpringBootIntegrationBaseTest {
+class OpenAPIPublisherTest extends SpringBootIntegrationBaseTest {
 
     @Autowired
     private MockMvc mvc;
 
     @DisplayName("Generate swagger documentation")
     @Test
     void generateDocs() throws Exception {
-        byte[] specs = mvc.perform(get("/v2/api-docs"))
+        byte[] specs = mvc.perform(get("/v3/api-docs"))
```

Comment:
Why has this version incremented ?

Reply:
as part of moving from sprinfox to springdoc, we are now using OpenApi/Swagger version3